tests/geometry/geometry_test.py

```diff
@@ -0,0 +1,43 @@
+# Copyright OTT-JAX
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+import jax
+import jax.numpy as jnp
+import numpy as np
+
+from ott.geometry import pointcloud
+
+
+@pytest.mark.fast()
+class TestCostStd:
+
+  def test_coststd(self, rng: jax.Array):
```

Comment:
I'd split this into 2 tests (or parameterize by `stat: Literal["mean", "std"]`.

Reply:
but then there would be lots of "if" no? on both computation of criterion and computation in geom?